Source/Laser/LaserEnvelope.cpp

```diff
@@ -0,0 +1,10 @@
+#include "LaserEnvelope.H"
+
+using namespace amrex;
```

Comment:
We need to define a constructor here.

Reply:
The constructor would do similar things as here:
https://github.com/ECP-WarpX/WarpX/blob/development/Source/Particles/LaserParticleContainer.cpp#L82

Reply:
In order to decide which parameters to read:
https://warpx.readthedocs.io/en/latest/usage/parameters.html#laser-initialization